source/module_psi/include/gpu_cuda.h

```diff
@@ -0,0 +1,121 @@
+#pragma once
+#include <vector>
+#include <stdio.h>
+#include <assert.h>
+#include <cuda_runtime.h>
+
+namespace gpu_cuda {
```

Comment:
Maybe cuda is enough?
```suggestion
namespace cuda {
```

Reply:
In order to distinguish from gpu_rocm